tools/cli-generator/templates/create.txt

```diff
@@ -0,0 +1,97 @@
+// Copyright {{ Year }} MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This code was autogenerated at {{ Now }}
```

Comment:
```suggestion
// This code was autogenerated at {{ Now }}. Note: Manual updates are allowed, but may be overwritten
```

Reply:
Also, good to include a link to a repo .MD about how the generator works.

Reply:
done